PublicUserRegistration/projects/PublicUserRegistration.json

```diff
@@ -1,18 +1,18 @@
 {
   "links" : [ ],
-  "name" : "PublicOnboardingDemo",
+  "name" : "PublicUserRegistration",
   "options" : {
-    "description" : "Public Onboarding Demo\n\nThis contribution allows public users to sign up for a VANTIQ account without requiring a VANTIQ employee to manually send them a link. This contribution only works in an Application Namespace and not in a Developer Namespace. \n\nThis namespace contains two clients, the first is publicly accessibly and the second is private, and one procedure. The first client allows the user to input their name and email and register for a VANTIQ account. The client makes sure that the user has filled out all the fields and has a valid email address. Then, the user is sent an email with a link to the VANTIQ sign up form with keycloak. Once they have gone through the keycloak sign up process, they are redirected to the second client which is a landing page that tells the user what their account credentials are. \n\n",
+    "description" : "Public User Registration Demo\n\nThis demo allows public users to sign up for a VANTIQ account without requiring a VANTIQ employee to manually send them a link. Please note, this registration process only works in an Application Namespace and not in a Developer Namespace. \n\nThis namespace contains two clients, the first is publicly accessibly (UserRegistration) and the second is private (LandingPage). The first client allows the user to input their name and email and register for a VANTIQ account. The client makes sure that the user has filled out all the fields and has a valid email address. \n\nWhen the user clicks the Register button, a public procedure is called, which generates an email invite. The email invite contains a link with a one time use code, which will grant the user access to VANTIQ. Before the code can be processed, the user will be redirected to keycloak, where they will identify themselves or create an account, and after which the code will be automatically processed and the user will be redirected to the private client. \n\nThe private client will simply show the username of the newly created identity in keycloak to confirm that the user is now logged in. \n",
```

Comment:
might use "This project" instead of "This namespace".   And "The public client" instead of "first client"
```suggestion
    "description" : "Public User Registration Demo\n\nThis demo allows public users to sign up for a VANTIQ account without requiring a VANTIQ employee to manually send them a link. Please note, this registration process only works in an Application Namespace and not in a Developer Namespace. \n\nThis project contains two clients, the first is publicly accessibly (UserRegistration) and the second is private (LandingPage). The public client allows the user to input their name and email and register for a VANTIQ account. The client makes sure that the user has filled out all the fields and has a valid email address. \n\nWhen the user clicks the Register button, a public procedure is called, which generates an email invite. The email invite contains a link with a one time use code, which will grant the user access to VANTIQ. Before the code can be processed, the user will be redirected to keycloak, where they will identify themselves or create an account, and after which the code will be automatically processed and the user will be redirected to the private client. \n\nThe private client will simply show the username of the newly created identity in keycloak to confirm that the user is now logged in. \n",
```